subsys/mgmt/mcumgr/Kconfig

```diff
@@ -15,6 +15,24 @@ module = MCUMGR
 module-str = mcumgr
 source "subsys/logging/Kconfig.template.log_config"
 
+config MCUMGR_ALIGNED_MEMORY_REQUIREMENT
+	bool
+	depends on MCUMGR_ALIGN_MEMORY
+	default y
```

Comment:
Shouldn't this be done by `select` in `MCUMGR_ALIGN_MEMEORY`?

Reply:
Fixed